test/dialect/postgresql/test_reflection.py

```diff
@@ -907,6 +907,74 @@ def go():
             ],
         )
 
+    @testing.fails_if("postgresql < 8.3", "index ordering not supported")
+    @testing.provide_metadata
+    def test_index_reflection_with_sorting(self):
+        """reflect indexes with sorting options set"""
+        metadata = self.metadata
+
+        t1 = Table(
+            "party",
+            metadata,
+            Column("id", String(10), nullable=False),
+            Column("name", String(20)),
+            Column("aname", String(20)),
+        )
+        metadata.create_all()
+
+        # check ASC, DESC options alone
```

Comment:
I'm sure this is imitating the other tests but i tend to try to use a newer pattern here:

```

with testing.db.connect() as conn:
    conn.execute("create index ...")
    conn.execute("create index ...")
```